index.bs

```diff
@@ -217,6 +217,8 @@ AV1 Sample Format {#sampleformat}
 For tracks using the [=AV1SampleEntry=], an <dfn>AV1 Sample</dfn> has the following constraints:
 - the sample data SHALL be a sequence of [=OBU=]s forming a [=Temporal Unit=],
 - each OBU SHALL follow the [=open_bitstream_unit=] [=Low Overhead Bitstream Format=] syntax as specified in [[!AV1]]. Each OBU SHALL have the [=obu_has_size_field=] set to 1 except for the last OBU in the sample, for which [=obu_has_size_field=] MAY be set to 0, in which case it is assumed to fill the remaining of the sample,
+
+NOTE: When extracting OBUs from an ISOBMFF file, and depending on the capabilities of the decoder processing these OBUs, ISOBMFF parsers may need to either set the [=obu_has_size_field=] to 1 for some OBUs if not already set, or use the length-delimited bitstream format as defined in Annex B of [=AV1=].
```

Comment:
Looks good, but I think we need to be excruciatingly clear. Add (to the end of the text here):

> When ISOBMFF parsers set [=obu_has_size_field=] to 1 the length field MUST be added to the OBU.

An OBU with [=obu_has_size_field=] set to 1, but without a length field would not be a valid OBU.

Reply:
And should the "may" in the NOTE be a "MAY"?